libfreeipmi/include/freeipmi/spec/ipmi-ipv6-address-status-spec.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Agreed.  What file should it go into?  None of the current ones jumped out at me. :/